actiontext/lib/action_text/system_test_helper.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module ActionText
+  module SystemTestHelper
+    # Locates a Trix editor and fills it in with the given HTML.
+    #
+    # The editor can be found by:
+    # * its +id+
+    # * its +placeholder+
+    # * its +aria-label+
+    # * the +name+ of its input
+    #
+    # Examples:
+    #
+    #   # <trix-editor id="message_content" ...></trix-editor>
+    #   fill_in_rich_text_area "message_content", with: "Hello <em>world!</em>"
+    #
+    #   # <trix-editor placeholder="Your message here" ...></trix-editor>
+    #   fill_in_rich_text_area "Your message here", with: "Hello <em>world!</em>"
+    #
+    #   # <trix-editor aria-label="Message content" ...></trix-editor>
+    #   fill_in_rich_text_area "Message content", with: "Hello <em>world!</em>"
+    #
+    #   # <input id="trix_input_1" name="message[content]" type="hidden">
+    #   # <trix-editor input="trix_input_1"></trix-editor>
+    #   fill_in_rich_text_area "message[content]", with: "Hello <em>world!</em>"
+    def fill_in_rich_text_area(locator, with:)
+      page.execute_script(<<~JS, find(:rich_text_area, locator).native, with.to_s)
```

Comment:
There should be no need to call `.native` here, Capybara will convert to the correct parameter for the driver, and with Capybara 3.2+ you can call execute_script on the element itself which will pass the element as the first argument

    page.find(:rich_text_area, locator).execute_script(<<~JS, with.to_s)
      ...